### ISSUE TITLE
C++ syntax highlighting: curly braces inside function call spanning multiple lines breaks highlighting.

### ISSUE THREAD

**Issue by harrism** [SublimeText-Markdown/MarkdownEditing#241]
This code highlights correctly:

``````
```c++
void test() {
    foo({1});
}
```
``````

This code causes the closing backticks to not be properly detected, and the code region continues past them.

``````
```c++
void test() {
    foo({1
        });
}
```
``````

Screenshot attached.  I'm using v2.1.0 on ST3, OS X.
![screen shot 2015-03-06 at 9 35 50 pm](https://cloud.githubusercontent.com/assets/783069/6523955/d1b76a5a-c448-11e4-9578-d971bb23908d.png)

**Comment by felixhao28** [comment]
It seems that this issue only occurs for C and C++ syntax. But MDE handles C/C++ just like any other language...

**Comment by s7726** [comment]
I'm seeing this issue as well.

**Comment by felixhao28** [comment]
Upcoming fix described in #484. Closing it for now. Feel free to reopen it if that solution is not sufficient.
